barebones/core_portme.h

```diff
@@ -27,21 +27,21 @@ Original Author: Shay Gal-on
 	Define to 1 if the platform supports floating point.
 */
 #ifndef HAS_FLOAT 
-#define HAS_FLOAT 1
```

Comment:
Define these in Makefile e.g. -DHAS_FLOAT=0, that way there are fewer differences with the original

Reply:
This whole file is kind of made to be edited by hand. I see what you mean by adding it in the makefile, but would we want to do that for all the define statements I modified?

Reply:
The nice thing about doing it in the Makefile is if we have, say, a BlackParrot config which has float and 1 which doesn't we can just flip a switch in the Makefile rather than editing the code to test both